### ISSUE TITLE
How about performance for flower dataset?

### ISSUE THREAD

**Issue by John1231983** [ikhlestov/vision_networks#13]
I am interested in flower dataset which is a small data set of a few thousand flower images spread across 5 labels: daisy, dandelion, roses, sunflowers, tulips. Could you write the python file for the flower dataset, besides cifar and svhn? I want to use your code to train it. Thanks

The script to download and separate into the train and validation folder is at https://github.com/tensorflow/models/blob/master/research/inception/inception/data/download_and_preprocess_flowers.sh

**Comment by ikhlestov** [comment]
Hi John!
Flowers dataset was not added to the source code, because main reason of this repo to reproduce network from the paper and provide source code for it. And there are no any metrics in the paper based on this dataset. If you want to train network with any other dataset that is not related to the initial implementation you may fork the repo and make all required changes by yourself.
